### ISSUE TITLE
registration.php file missing

### ISSUE THREAD

**Issue by bpoiss** [zoransalamun/magento2-custom-controllers#1]
The installation as described is not working because of the missing registration file.

**Comment by zoransalamun** [comment]
Yes, module was created before registration.php implementation in Magento 2. I will update in week or two...
